ir/instr.cpp

```diff
@@ -2403,6 +2403,110 @@ unique_ptr<Instr> Memcpy::dup(const string &suffix) const {
 }
 
 
+
+static pair<expr,expr>
+encode_memcmp(State &s, const Pointer &p1, const Pointer &p2, AndExpr &prefix,
```

Comment:
isn't it possible to refactor this to share the code with strlen()? That was the original plan I guess. Have a generic way to implement these loopy intrinsics.

Reply:
That makes sense - we have prefix, and i, and maximum unroll count. I will work on it.